### ISSUE TITLE
Spot Check: Rutherford Scattering 1.0.0-dev.8

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/tasks#639]
Rutherford Scattering is ready for a spot check for the performance issues that came out of the last dev test.  This is not meant to be a full dev test, but just a check to make sure that performance is OK and that the bugs have been fixed.  Please test the sim on a few tablets and slower devices, and if you notice any issues that are new in this dev version, create a new issue and reference this one.

Dev version: http://www.colorado.edu/physics/phet/dev/html/rutherford-scattering/1.0.0-dev.8/rutherford-scattering_en.html

Please verify the following:
- [x]  All performance issues described in https://github.com/phetsims/rutherford-scattering/issues/71 are now satisfactory
- [x] https://github.com/phetsims/rutherford-scattering/issues/74 has been fixed
- [x] https://github.com/phetsims/rutherford-scattering/issues/70 has been fixed

**Comment by ariel-phet** [comment]
@phet-steele function builder is obviously top priority, but if these checks could be done in parallel it would be great so we can move rutherford to RC status

**Comment by alynne-phet** [comment]
@jessegreenberg I am still seeing alpha particles disappear when they hit the nucleus

**Comment by phet-steele** [comment]
@jessegreenberg performance checked, done. Only phetsims/rutherford-scattering/issues/74 remains.
